src/overlays/actors/ovl_En_Bom_Chu/z_en_bom_chu.c

```diff
@@ -336,22 +359,26 @@ void EnBomChu_WaitForKill(EnBomChu* this, GlobalContext* globalCtx) {
     }
 }
 
-void func_809C649C(EnBomChu* this, Vec3f* arg1, Vec3f* dest) {
-    f32 scale = arg1->x + this->unk_178;
-
-    dest->x =
-        this->actor.world.pos.x + (this->unk_16C.x * scale) + (this->unk_160.x * arg1->y) + (this->unk_154.x * arg1->z);
-    dest->y =
-        this->actor.world.pos.y + (this->unk_16C.y * scale) + (this->unk_160.y * arg1->y) + (this->unk_154.y * arg1->z);
-    dest->z =
-        this->actor.world.pos.z + (this->unk_16C.z * scale) + (this->unk_160.z * arg1->y) + (this->unk_154.z * arg1->z);
+/*
+ * Transform coordinates from model space to world space, according to current orientation.
+ * `posModel` is expected to already be at world scale (1/100 compared to model scale)
+ */
```

Comment:
the function level comments get two stars at the opening comment /**
this is for the doxygen generation, and most text editors will know to automatically place a star on each new line when there is two